SQL Datasources: Update Max Connection and Max Idle Connection Defaults to 100 and add auto mode (#65834)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

<hr />

## [sql_datasources]

### max_open_conns_default

> For SQL datasources (MySql, Postgres, MSSQL) override the default maximum number of open connections (default: 100). Values configured in data source settings will be preferred over this value.

Reviewer: ```suggestion
For SQL data sources (MySql, Postgres, MSSQL) you can override the maximum number of open connections (default: 100).  The values configured in data source settings will be preferred over this value.
```

### max_idle_conns_default

> For SQL datasources (MySql, Postgres, MSSQL) override the default allowed number of idle connections (default: 100). Values configured in data source settings will be preferred over this value.

Reviewer: ```suggestion
For SQL data sources (MySql, Postgres, MSSQL) you can override the default allowed number of idle connections (default: 100). The values configured in data source settings will be preferred over the default value.
```

### max_conn_lifetime_default

> For SQL datasources (MySql, Postgres, MSSQL) override the default maximum connection lifetime specified in seconds (default: 14400). Values configured in data source settings will be preferred over this value.

Reviewer: ```suggestion
For SQL data sources (MySql, Postgres, MSSQL) you can override the default maximum connection lifetime specified in seconds (default: 14400). The values configured in data source settings will be preferred over the default value.
```
